controllers/reservation.controllers.js

```diff
@@ -457,22 +462,12 @@ const submitReserve = async ({
     const roomFull = await verifyRoomFull(_day, user.office, room.value);
 
     if (repeteadDay) {
-      errorObject.errors = {
-        date_input: `No es posible agendarte, ya tienes una reserva para el día ${moment(
-          _day,
-        ).format('DD/MM/YYYY')}`,
-      };
-      errors = errorObject;
       return;
     }
 
     if (roomFull) {
-      errorObject.errors = {
-        date_input: `No es posible agendarte, el día ${moment(_day).format(
-          'DD/MM/YYYY',
-        )} tiene la sala llena`,
-      };
-      errors = errorObject;
+      const dayFormated = `${moment(_day).format('DD/MM/YYYY')}`;
```

Comment:
Is it necessary to wrap it in strings? I believe the return value is already a string